roles/oneagent/tests/README.md

```diff
@@ -43,11 +46,34 @@ $ pip install -r roles/oneagent/tests/requirements.txt
 $ mkdir -p roles/oneagent/files && wget https://ca.dynatrace.com/dt-root.cert.pem -P roles/oneagent/files
 $ ansible-galaxy collection build . -vvv
 $ sudo bash -c "source venv/bin/activate && ansible-galaxy collection install -vvv dynatrace-oneagent*"
+```
 
+### Running tests locally and remotely
+
+Running tests with regular manner requires one of the following commands:
+
+```commandline
```

Comment:
You can move the `#` comments out of the `commandline` sections.

Reply:
I prefer it in this way